### PR TITLE
adding notes to prevent Libre admins asking permissions

### PR DESCRIPTION
**Pull request by orangesunny**
several Libre project admins asked about the permissions, so I made the documentation clearer for them

Before submitting a pull request, please ensure that:

- [x] Every commit has a message which concisely describes it.
- [x] Every commit is needed on its own; if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by user documentation.

### CODE REVIEW

File: docs/admin/access.rst [hunk 1]
```diff
@@ -41,12 +41,20 @@ all projects configured as `Public` (see :ref:`acl`).
 Per project permissions
 +++++++++++++++++++++++
 
+.. note::
+ 
+    This feature is not available for projects running Hosted Libre plan.
```
Comment (comradekingu) [inline docs/admin/access.rst:46]:
```suggestion
    This feature is not available for projects running the Hosted Libre plan.
```

File: docs/admin/access.rst [hunk 2]
```diff
@@ -41,12 +41,20 @@ all projects configured as `Public` (see :ref:`acl`).
...
+
 Set your projects to `Protected` or `Private`, and manage users per
 project in the Weblate interface.
 
 Adding permissions to languages, components or projects
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+.. note::
+ 
+    This feature is not available for projects running Hosted Libre plan.
```
Comment (comradekingu) [inline docs/admin/access.rst:56]:
```suggestion
    This feature is not available for projects running the Hosted Libre plan.
```

File: docs/admin/access.rst [hunk 3]
```diff
@@ -71,6 +79,10 @@ Per project access control
     By enabling ACL, all users are prohibited from accessing anything within a given
     project, unless you add the permissions for them to do just that.
 
+.. note::
+ 
+    This feature is not available for projects running Hosted Libre plan.
```
Comment (comradekingu) [inline docs/admin/access.rst:84]:
```suggestion
    This feature is not available for projects running the Hosted Libre plan.
```

File: docs/admin/access.rst (hunk 1 above)
Comment (orangesunny) [reply docs/admin/access.rst:46]: thanks

File: docs/admin/access.rst [hunk 4]
```diff
@@ -41,12 +41,20 @@ all projects configured as `Public` (see :ref:`acl`).
 Per project permissions
 +++++++++++++++++++++++
 
+.. note::
+ 
+    This feature is not available for projects running the Hosted Libre plan.
+
```
Comment (nijel) [inline docs/admin/access.rst:47]:
```suggestion
.. note::

    This feature is not available for projects running the Hosted Libre plan.

```

File: docs/admin/access.rst [hunk 5]
```diff
@@ -41,12 +41,20 @@ all projects configured as `Public` (see :ref:`acl`).
...
 Set your projects to `Protected` or `Private`, and manage users per
 project in the Weblate interface.
 
 Adding permissions to languages, components or projects
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+.. note::
+ 
+    This feature is not available for projects running the Hosted Libre plan.
+
```
Comment (nijel) [inline docs/admin/access.rst:57]:
```suggestion
.. note::

    This feature is not available for projects running the Hosted Libre plan.

```

File: docs/admin/access.rst [hunk 6]
```diff
@@ -71,6 +79,10 @@ Per project access control
     By enabling ACL, all users are prohibited from accessing anything within a given
     project, unless you add the permissions for them to do just that.
 
+.. note::
+ 
+    This feature is not available for projects running the Hosted Libre plan.
+
```
Comment (nijel) [inline docs/admin/access.rst:85]:
```suggestion
.. note::

    This feature is not available for projects running the Hosted Libre plan.

```

**Review by nijel — COMMENTED** [review]
Remove extra whitespace to get rid of pre-commit complains.

**Pull request merged**